docs/Building-Transmission.md

```diff
@@ -39,6 +39,41 @@ ninja -C build transmission-gtk
 ## On Unix ##
 ### Prerequisites ###
 
+#### Debian 12 / Bookworm ####
+On Debian, you can build transmission with a few dependencies on top of a base installation.
+
+For building transmission-daemon you will need basic dependencies:
+```console
+$ sudo apt install build-essential cmake git libcurl4-openssl-dev libssl-dev
+```
+These packages are not mandatory for a working binary. Transmission brings its own libraries if they aren't installed.
+```console
+$ sudo apt install libb64-dev libdeflate-dev libevent-dev libminiupnpc-dev libnatpmp-dev libpsl-dev libsystemd-dev
+```
+
+You likely want to install transmission as a native GUI application.
+There are two options, GTK and QT. Both of them have 2 versions.
```

Comment:
In my dictionary, QT usually stands for QuickTime, while the framework you want to reference is spelled Qt (with lowercase "t"). Same down below, and in other sections that you didn't author...

Reply:
I might be the `git blame` for them as well :cold_sweat: